projects/unicorn/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER unicorn.emu@gmail.com
+RUN apt-get update && apt-get install -y make python
+#TODO change to main repo
+RUN git clone --depth 1 --branch fuzz https://github.com/catenacyber/unicorn.git
```

Comment:
This needs to be the original unicorn repo, if there are custom patches, please add a diff file and then apply in build.sh